src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -69,10 +91,11 @@ public AttachCaseCallbackService(CcdApi ccdApi, PaymentsProcessor paymentsProces
      *         or the list of errors, in case of errors
      */
     public Either<List<String>, Map<String, Object>> process(
-        CaseDetails exceptionRecord,
+        CaseDetails exceptionRecordData,
```

Comment:
If we're making a change from `exceptionRecord`, how about `exceptionRecordDetails`? `CaseDetails` contains `data`, so this might be slightly confusing.

Reply:
Will rename